charts/ditto-digital-twins/README.md

```diff
@@ -77,3 +77,18 @@ gateway:
   systemProps:
     - "-Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider=openid-connect.onelogin.com/oidc"
 ```
+
+### Securing Devops Resource
+
+To secure /devops and /status resource adjust configuration to (username will be `devops`):
+
+```yaml
+gateway:
+  enableDummyAuth: false
+  devopsSecureStatus: true
+  extraEnv:
+    - name: DEVOPS_PASSWORD
+      value: foo
+    - name: STATUS_PASSWORD
+      value: bar
+``
```

Comment:
can we put passwords in secrets please?

Reply:
This is just an example.

Reply:
Let us improve this in an upcoming PR

Reply:
this is also wrong for an example, as it will be used this way.